client/web/src/sveltekit/SvelteKitNavItem.tsx

```diff
@@ -16,17 +17,17 @@ export const SvelteKitNavItem: FC<{ userID?: string }> = ({ userID }) => {
     const location = useLocation()
     const client = useApolloClient()
     const [departureDismissed, setDepartureDismissed] = useTemporarySetting('webNext.departureMessage.dismissed', false)
-    const [welcomeDismissed, setWelcomeDismissed] = useTemporarySetting('webNext.welcomeOverlay.dismissed', false)
+    const [showDeparture, _setDepartureDismissed] = useTemporarySetting('webNext.departureMessage.show', false)
+    const [_showWelcomeMessage, setShowWelcomeMessage] = useTemporarySetting('webNext.welcomeOverlay.show', false)
```

Comment:
FWIW in array destructuring you can omit values if you don't want to use them. I.e. you can could do

```suggestion
    const [showDeparture] = useTemporarySetting('webNext.departureMessage.show', false)
    const [, setShowWelcomeMessage] = useTemporarySetting('webNext.welcomeOverlay.show', false)
```

(though I don't if our lint rules allow the second one)

Reply:
`[, setShowWelcomeMessage]` will be errored by the linters, I think.